src/openms/source/SYSTEM/RWrapper.cpp

```diff
@@ -46,62 +46,97 @@
 namespace OpenMS
 {
 
-  bool RWrapper::runScript( const String& script_file, const QStringList& cmd_args )
+  bool RWrapper::runScript( const String& script_file, const QStringList& cmd_args, bool find_R /*= false */, bool verbose /*= true */)
   {
+    if (find_R && !findR(verbose))
+    {
+      return false;
+    }
+    
     String fullscript;
     try
     {
-      fullscript = findScript(script_file);
+      fullscript = findScript(script_file, verbose);
     }
     catch (...)
     {
       return false;
     }
 
-    LOG_INFO << "Running R script '" << fullscript << "' ..." << std::flush;
-    if (!findR())
-    {
-      return false;
-    }
-
-    QProcess p;
-    p.setProcessChannelMode(QProcess::MergedChannels);
+    if (verbose) LOG_INFO << "Running R script '" << fullscript << "' ...";
```

Comment:
I'm generally not a fan of the space before "..." (here and elsewhere), but that may be a matter of (typographic) taste.